rust/gelato/src/task_status.rs

```diff
@@ -67,7 +67,7 @@ pub struct TaskStatusApiCall {
 impl TaskStatusApiCall {
     #[instrument]
     pub async fn run(&self) -> eyre::Result<TaskStatusApiCallResult> {
-        let url = format!("{}/tasks/status/{}", RELAY_URL, self.args.task_id);
+        let url = format!("{RELAY_URL}/tasks/status/{}", self.args.task_id);
```

Comment:
For my own understanding, why can't we put self.args.task_id inside {} as well?

Reply:
It has to be an [`Ident`](https://doc.rust-lang.org/reference/identifiers.html).